css/mistica-common.css

```
@@ -938,3 +938,68 @@ button.mistica-display-card:has(.mistica-card__media):active:after {
 .mistica-accordion-item[open]::details-content {
     height: auto;
 }
+
+/* Column Layout */
```

```suggestion
/* Grid Layout */
```
To follow the same naming we use in react

---

Renamed